introduction.md

```diff
@@ -1,3 +1,11 @@
 This is my intro to the codeschool mastering github
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?